projects/chakra/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -eu
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+DISABLED_WARNINGS='-Wno-everything'
+
+# Inject additional C/CXXFLAGS
```

Comment:
This looks weird :) but I guess there is no better option?

Maybe `--extra-defines` can be useful: https://github.com/Microsoft/ChakraCore/blob/master/build.sh#L44 ?

Reply:
I don't think that can be used to set CMAKE_C(XX)FLAGS :(